src/shared/utils/forms.ts

```diff
@@ -0,0 +1,9 @@
+import i18n from 'i18n';
+
+export const getDictionaryText = (description?: string | Record<string, string>) => {
+  if (!description) return '';
+
+  const { language } = i18n;
+
+  return typeof description === 'object' ? description[language] ?? '' : description ?? '';
```

Comment:
if this way?
  return (typeof description === 'object' ? description[language]  : description) ?? ""